<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.13.0 on win32)" generated="2024-12-07T00:41:54.394929" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\souza\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2024-12-07T00:41:54.444640" elapsed="46338.343269"/>
</kw>
<status status="PASS" start="2024-12-07T00:41:54.444054" elapsed="46338.357133"/>
</test>
<status status="PASS" start="2024-12-07T00:41:54.396706" elapsed="46338.408083"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2024-12-07T00:41:54.382907" level="WARN">Error in file 'c:\Users\souza\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
